src/server.js

```diff
@@ -0,0 +1,38 @@
+const Hapi = require("hapi")
+
+const server = (cluster) => {
+  const server = new Hapi.Server()
+
+  server.connection({ port: process.env.PORT || 8080 })
+
+  server.route({
+    method: "GET",
+    path: "/",
+    handler: (request, reply) => {
+      const response = reply('Server running')
+      response.type('text/plain')
+      response.statusCode = 200
+    }
+  })
+
+  server.route({
+    method: "GET",
```

Comment:
I feel like having this route as a GET is confusing, since it's purpose is to stop the build? Should it be a DELETE? Also, are we actually registering a callback in this route?